src/clib/grackle_chemistry_data.h

```diff
@@ -136,6 +136,21 @@ typedef struct
   /* flag for Wolcott-Green+ 2011 H2 self-shielding */
   int H2_self_shielding;
 
+  /* flag to select which formula for calculating k11 you want to use. Setting to 1 will use Savin 2004, 0 will use Abel et al. 1997  */
```

Comment:
The other new parameters (defined below) seem to use 1 or 2 rather than 0 or 1. Any particular reason for this choice? Naively, it seems like 1 or 2 for selecting between two options makes more sense to me (with 0 reserved for the option of turning it off entirely).

Reply:
You make a good point about the arbitrarity of my conventions here. I will change these to be more immediate / intuitive for the user. Thanks for all of your comments!